main/manager.c

```diff
@@ -1728,6 +1728,7 @@ struct mansession_session {
 	time_t noncetime;	/*!< Timer for nonce value expiration */
 	unsigned long oldnonce;	/*!< Stale nonce value */
 	unsigned long nc;	/*!< incremental  nonce counter */
+	unsigned int kicked:1;	/*!< Flag set if session is forcibly kicked */
```

Comment:
Should this be set to 0 in `build_mansession`?

Reply:
It gets allocated zeroed out.